code/src/containers/forms/new-product-form/NewProductForm.scss

```diff
@@ -0,0 +1,133 @@
+@import "@design-system";
+
+.product-form {
+  display: grid;
+  grid-template-columns: calc(30% - 10px) calc(70% - 10px);
+  grid-gap: spacing(5);
+  margin-bottom: spacing(5);
+
+  &__header {
+    background-color: var(--new-product-form-head-bg-color, $light-alpha);
+    padding: spacing(5) spacing(8);
+  }
+
+  &__body {
+    padding: spacing(8);
+  }
+
+  &__container {
+    box-shadow: var(--new-product-form-container-shadow, $shadow-black);
+    background-color: var(--new-product-form-container-bg-color, $white);
+  }
+
+  &__image-preview {
+    border: 1px solid var(--spa-accordion-container-border-color, $gray-light);
+    border-radius: 3px;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    margin-bottom: spacing(5);
+    aspect-ratio: 1 / 1;
+  }
+
+  &__image-section {
+    grid-row: 1 / 2;
+    grid-column: 1 / 2;
+  }
+
+  &__main-info-section {
+    grid-column: 2 / 3;
+    grid-row: 1 / 3;
+    display: flex;
+    flex-direction: column;
+    gap: spacing(5);
+
+    & > div {
+      flex: 3 0 auto;
+    }
+  }
+
+  &__additional-info-section {
+    grid-column: 1 / 2;
+    grid-row: 2 / 3;
+
+    .product-form__body {
+      padding-bottom: spaacing(3);
+    }
+  }
+
+  &__category-select {
+    padding: spacing(3) spacing(11) spacing(3) spacing(14) spacing(3);
+
+    &-label {
+      display: inline;
+    }
+  }
+  &__quantity-price-container {
+    display: flex;
+    gap: spacing(5);
+    margin-bottom: spacing(5);
+  }
+
+  &__text-input {
+    margin-bottom: spacing(4);
+
+    &:last-child {
+      margin-bottom: 0;
+    }
+  }
+
+  &__description-input {
+    padding: 1.75px 1px !important; // To rewrite mui styles
+  }
+
+  &__header-title {
+    font-size: $fs-base;
+    font-weight: bold;
+    color: var(--new-product-form-header-title-color, $white);
+  }
+
+  &__image-preview-text {
+    color: var(--spa-text-color, $gray);
+    text-align: center;
+  }
+
+  &__image {
+    @include box(160px);
+
+    object-fit: cover;
+  }
+
+  &__footer {
+    display: grid;
+    grid-template-columns: calc(30% - 10px) calc(70% - 10px);
```

Comment:
Good use case to use local css variable because i hope `10px` is repeated. Maybe also use `spacing` here?

Reply:
Used variable